spec/system/posts/posts_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
```

Comment:
- [ ] Please make sure that all tests run, as there are some tests not run for the Post.
![image](https://user-images.githubusercontent.com/24703171/181070256-ffad976a-f9e9-4515-9e1f-ed92040f4187.png)

Reply:
@gemmen29, check again all the tests are working 

![Screenshot_20220726_183730](https://user-images.githubusercontent.com/25227022/181074049-22bcd0ef-83db-4353-ab58-fed31aace006.png)